proxy/coco/src/project/create/validation.rs

```diff
@@ -375,6 +375,14 @@ impl Repository {
         url: &LocalUrl,
     ) -> Result<Option<Remote<LocalUrl>>, Error> {
         match Remote::<LocalUrl>::find(repo, reflike!("rad")) {
+            Err(remote::FindError::ParseUrl(_)) => {
+                log::warn!("an old/invalid URL was found when trying to load the `rad` remote");
+                log::warn!(
+                    "we are going to rename the remote to `temp_rad` and create a new `rad` remote"
+                );
+                repo.remote_rename("rad", "temp_rad")?;
```

Comment:
It's not temporary is it? Perhaps `rad_old`?

Reply:
It is if they remove it later :smile: `rad_old` works too :+1: